pyproject.toml

```
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: Helmholtz Centre for Environmental Research (UFZ)
+# SPDX-FileCopyrightText: Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[project]
+name = "ansible-collection-toolkit"
+version = "5.3.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.11"
+dependencies = [
+    "ansible>=11.3.0",
+]
+
```

I wonder if this is something we should take into account:
```suggestion

[tool.uv]
package = false

```
See [[1]](https://docs.astral.sh/uv/concepts/projects/config/#project-packaging) and [[2]](https://docs.astral.sh/uv/reference/settings/#package) (Similar to poetry's [`package-mode`](https://python-poetry.org/docs/pyproject/#package-mode)).

---

I did not add it since the docs say

> While uv usually uses the declaration of a [build system](https://docs.astral.sh/uv/concepts/projects/config/#build-systems) to determine if a project should be packaged, uv also allows overriding this behavior with the [tool.uv.package](https://docs.astral.sh/uv/reference/settings/#package) setting.

This is why I did not add this variable as it's implicit by not providing the `build-system`.